src/tests/system/tests/test_ldap.py

```diff
@@ -14,173 +14,102 @@
+    modify_mode: str
 ):
     """
-    :title: Password change using ppolicy
-    :description: PPolicy overlay is the latest implementation of IETF password policy for LDAP.
-    This extends the password policy for the LDAP server and is configured in SSSD using
-    'ldap_use_ppolicy'.
-
-    Two password modification modes are tested, Extended Operation (exop), the default and then
-    LDAP (ldapmodify), set by 'ldap_pwmodify_mode' parameter.
-    :note: This feature is introduced in SSSD 2.10.0
+    :title: User logins and issues a password change
     :setup:
-        1. Add a user to LDAP
-        2. Configure the LDAP ACI to permit user password changes
-        3. Set "ldap_pwmodify_mode"
-        4. Start SSSD
+        1. Create user 'user'
+        2. Configure SSSD with 'ldap_pwmodify_mode = exop | ldap_modify | exop_force' and 'ldap_user_ppolicy = true
+        3. Start SSSD
     :steps:
-        1. Authenticate as user
-        2. Change the password of user
-        3. Authenticate user with new password
-        4. Authenticate user with old password
+        1. Login as user
+        2. Issue password change and enter a bad confirmation password
+        3. Issue password change and enter a good confirmation password
+        4. Login with old password
+        5. Login with new password
     :expectedresults:
         1. User is authenticated
-        2. Password is changed successfully
-        3. User is authenticated
-        4. User is not authenticated
+        2. Password is change is unsuccessful
+        3. Password change  is successful
+        4. User cannot log in
+        5. User can log in
     :customerscenario: True
     """
-    user = "user1"
-    old_pass = "Secret123"
-    new_pass = "New_password123"
-
-    ldap.user(user).add(password=old_pass)
-    ldap.aci.add('(targetattr="userpassword")(version 3.0; acl "pwp test"; allow (all) userdn="ldap:///self";)')
+    old_password = "Secret123"
+    invalid_password = "secret"
+    new_password = "Secret123**%%"
 
+    ldap.user("user1").add(password = old_password)
     client.sssd.domain["ldap_pwmodify_mode"] = modify_mode
-    client.sssd.domain["ldap_use_ppolicy"] = use_ppolicy
-    client.sssd.start(service_user=sssd_service_user)
-
-    assert client.auth.ssh.password(user, old_pass), "Login with old password failed!"
-
-    assert client.auth.passwd.password(user, old_pass, new_pass), "Password change failed!"
-
-    assert client.auth.ssh.password(user, new_pass), "User login failed!"
-    assert not client.auth.ssh.password(user, old_pass), "Login with old password passed!"
-
-
-@pytest.mark.ticket(bz=[795044, 1695574])
-@pytest.mark.importance("critical")
-@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify", "exop_force"])
-@pytest.mark.parametrize("use_ppolicy", ["true", "false"])
-@pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.builtwith("ldap_use_ppolicy")
-def test_ldap__password_change_new_passwords_do_not_match_using_ppolicy(
```

Comment:
This one was just merged into the above test case right?

Reply:
Yea, it's an added step in most of the tests. 

```
        2. Issue password change and enter a bad confirmation password
```